src/test/java/api/requests/InstanceRequestsAPICreationTests.java

```diff
@@ -482,14 +724,14 @@ private void validateInstanceRequestResponse(JsonObject representation,
   private JsonObject createInstanceRequestObject(UUID instanceId, UUID requesterId, UUID pickupServicePointId,
                                                  DateTime requestDate, DateTime requestExpirationDate){
     JsonObject requestBody = new JsonObject();
-    requestBody.put("instanceId", instanceId.toString());
-    requestBody.put("requestDate", requestDate.toString(ISODateTimeFormat.dateTime()));
-    requestBody.put("requesterId", requesterId.toString());
-    requestBody.put("pickupServicePointId", pickupServicePointId.toString());
-    requestBody.put("fulfilmentPreference", "Hold Shelf");
-    if (requestExpirationDate != null) {
-      requestBody.put("requestExpirationDate", requestExpirationDate.toString(ISODateTimeFormat.dateTime()));
-    }
+    write(requestBody,"instanceId", instanceId.toString());
```

Comment:
```suggestion
    write(requestBody, "instanceId", instanceId.toString());
```

Same with below `write`s.

Reply:
Done